auslib/web/admin/views/validators.py

```diff
@@ -124,24 +93,23 @@ def priority_validator(field_value):
     try:
         x = int(field_value)
     except:
-        raise JsonschemaValidationError('Invalid input %s for priority.Not an integer.' % field_value)
-    if x < 0:
-        raise JsonschemaValidationError("Priority field value %s shouldn't be less than 0" % field_value)
+        raise JsonschemaValidationError(message=("Invalid input for %s. Not an integer." % field_name))
+    if min_val is not None and x < min_val:
+        raise JsonschemaValidationError(
+            message=("%s field value should be an integer >= %s" % field_name, min_val))
```

Comment:
I think you need () around the formatting parameters here - this is currently causing 500s like:
balrogadmin_1  |   File "./auslib/web/admin/views/validators.py", line 99, in integer_and_range_validator
balrogadmin_1  |     message=("%s field value should be an integer >= %s" % field_name, min_val))
balrogadmin_1  | TypeError: not enough arguments for format string


Looks fine other than that.